### PR TITLE
Samsung Internet, Updating A-G files, in api/

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
A checklist to help your pull request get merged faster:
- [ ] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

**Comment by vinyldarkscratch** [comment]
Looking good, thank you!  There's a few conflicts between this PR and the results of the mirroring script I wrote, but for some reason GitHub's not letting me send a review in.  I'll try again tomorrow, but overall it's looking great!

**Comment by AdaRoseCannon** [comment]
Feedback addressed.

File: api/CSS.json [hunk 1]
```diff
@@ -1545,9 +1545,16 @@
...
-            "samsunginternet_android": {
-              "version_added": "4.0"
-            },
+            "samsunginternet_android": [
+              {
+                "version_added": "8.0"
+              },
+              {
+                "partial_implementation": true,
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/CSS.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Document.json [hunk 2]
```diff
@@ -943,6 +986,20 @@
...
               }
             ],
+            "samsunginternet_android": [
+              {
+                "version_added": "5.0"
+              },
+              {
+                "alternative_name": "charset",
+                "version_added": true,
+                "notes": "<code>charset</code> alias was made read-only in 45."
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
                "notes": "<code>charset</code> alias was made read-only in Samsung Internet 5.0."
```

File: api/Document.json [hunk 3]
```diff
@@ -1761,6 +1863,10 @@
               "safari_ios": {
                 "version_added": null
               },
+              "samsunginternet_android": {
+                "version_added": false,
```
Comment (vinyldarkscratch) [inline api/Document.json]: I think this is true and Chrome Android is falsely marked as...well, `false`, considering Chrome, Opera Android, and WebView Android all have support.  I'm opening up a PR to resolve Chrome Android!

```suggestion
                "version_added": true,
```

File: api/Document.json [hunk 4]
```diff
@@ -2186,6 +2316,16 @@
             "safari_ios": {
               "version_added": true
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
                "version_added": "1.5",
```

File: api/Document.json [hunk 5]
```diff
@@ -2243,6 +2383,10 @@
             "safari_ios": {
               "version_added": true
             },
+            "samsunginternet_android": {
+              "version_added": "2.0",
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
              "version_added": "1.5",
```

File: api/Document.json [hunk 6]
```diff
@@ -2426,6 +2579,9 @@
             "safari_ios": {
               "version_added": "8"
             },
+            "samsunginternet_android": {
+              "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
              "version_added": "2.0"
```

File: api/Document.json [hunk 7]
```diff
@@ -4065,6 +4293,15 @@
             "safari_ios": {
               "version_added": null
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "5.0"
+              },
+              {
+                "prefix": "webkit",
+                "version_added": true
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
                "version_added": "1.5"
```

File: api/Document.json [hunk 8]
```diff
@@ -4515,6 +4765,10 @@
             "safari_ios": {
               "version_added": null
             },
+            "samsunginternet_android": {
+              "alternative_name": "webkitIsFullScreen",
```
Comment (vinyldarkscratch) [inline api/Document.json]: Another weird issue in the original data...  Filing a PR for this for Chrome, but if `version_added` is `false`, it's impossible for it to have an alternate name.

```suggestion
```

File: api/Document.json [hunk 9]
```diff
@@ -5409,6 +5711,15 @@
             "safari_ios": {
               "version_added": "7"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
                "version_added": "2.0"
```

File: api/Document.json [hunk 10]
```diff
@@ -5409,6 +5711,15 @@
...
+              },
+              {
+                "prefix": "webkit",
+                "version_added": true
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
                "version_added": "1.0"
```

File: api/Document.json [hunk 11]
```diff
@@ -6951,6 +7339,9 @@
             "safari_ios": {
               "version_added": "7"
             },
+            "samsunginternet_android": {
+              "version_added": "3.0"
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
              "version_added": "2.0"
```

File: api/Document.json [hunk 12]
```diff
@@ -8624,6 +9080,9 @@
             "safari_ios": {
               "version_added": false
             },
+            "samsunginternet_android": {
+              "version_added": "4.0"
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
              "version_added": "3.0"
```

File: api/Document.json [hunk 13]
```diff
@@ -9849,6 +10345,15 @@
             "safari_ios": {
               "version_added": "7"
             },
+            "samsunginternet_android": [
+              {
+                "version_added": "3.0"
+              },
+              {
+                "prefix": "webkit",
+                "version_added": true
```
Comment (vinyldarkscratch) [inline api/Document.json]:
```suggestion
                "version_added": "1.0"
```

File: api/Element.json [hunk 14]
```diff
@@ -7730,7 +7772,7 @@
               "version_added": true
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "2.0"
```
Comment (vinyldarkscratch) [inline api/Element.json]:
```suggestion
              "version_added": "1.5"
```

**Pull request merged**